docs/mergeduty/howto.md

```diff
@@ -122,9 +122,12 @@ There are ftl check hooks on [hg.m.o](http://hg.mozilla.org/) that prevent users
 1. The diff for `release` should be fairly similar to [this](https://hg.mozilla.org/releases/mozilla-release/rev/70e32e6bf15e), with updated branding as well as the version change.
 1. Push your changes generated by no-op trial run:
 ```sh
+# Set this variable to your ldap e-mail address to make sure
```

Comment:
I think we need to add these instructions at lines 89, 96 and 103 too.

Reply:
Will do.